Resources/Prototypes/Entities/Objects/Misc/utensils.yml

```
@@ -55,7 +57,7 @@
   parent: UtensilBase
   id: SpoonPlastic
   name: plastic spoon
-  description: There is no spoon.
+  description: A fragile plastic spoon that will inevitably break.
```

```suggestion
  description: There is no spoon.
```

(Old line was better)